### PR TITLE
Exercise 9

### PR DESCRIPTION
**Pull request by vaibhav1sh**
Exercise 9 complete, all tests working and app is available at https://todo-app-devops.herokuapp.com/

### CODE REVIEW

File: tests_e2e/test_e2e.py [hunk 1]
```diff
@@ -40,23 +42,29 @@ def driver():
...
 def test_task_journey(driver, test_app):
     driver.get('http://localhost:5000/')
+    print('Page Title in task journey ==>', driver.title)
+    driver.implicitly_wait(10)
     assert driver.title == 'To-Do App'
-
+    
 def test_create_card(driver, test_app):
     driver.get('http://localhost:5000/')
+    driver.implicitly_wait(10)
```
Comment (BenRamchandani) [inline tests_e2e/test_e2e.py:51]: If you set this once in your fixture you shouldn't need to do it again

File: todo_app/trello_api_calls.py [hunk 2]
```diff
@@ -21,92 +21,66 @@ def __str__(self):
...
-def get_list_id(list_category):  
-    board_id = get_board_id()
-    api_suffix_get_lists = '1/boards/' + board_id + '/lists'
-    api_response_list = call_api(api_suffix_get_lists)
-    for x in api_response_list:
-        if x["name"] == list_category:
-            list_id = x["id"]
-    log.debug("get_list_id() gives list id:%s", list_id)
-    return list_id
+def establish_mongo_connection(mongo_db_name, mongo_collection_name):
```
Comment (BenRamchandani) [inline todo_app/trello_api_calls.py:45]: Maybe rename these arguments? e.g. `mongo_db_name_env_var`

File: .env.template [hunk 3]
```diff
@@ -22,4 +22,17 @@ TRELLO_BOARD_NAME_CREATE = board_to_be_created
 LOG_LEVEL=DEBUG
```
Comment (BenRamchandani) [inline .env.template:10]: You can remove the Trello env variables from here

**Review by BenRamchandani — APPROVED** [review]
Looks good :+1:

I've left a few optional comments, and it would be good to do some renaming to get rid of references to Trello. Feel free to merge when you're ready though.

File: todo_app/trello_api_calls.py (hunk 2 above)
Comment (vaibhav1sh) [reply todo_app/trello_api_calls.py:45]: Sure

**Pull request merged**